liquid.py

```diff
@@ -256,12 +290,45 @@ def alloc(job, group):
     print(first(running, 'running allocations'))
 
 
-def nomad_address():
+def get_search_collections():
+    try:
+        return run(prepare_command('hoover-search', './manage.py', 'listcollections'), shell=False).split()
```

Comment:
Let's extract this bit as a function:
```python
def docker_exec(liquid_task_name, *args):
    return run(prepare_command(liquid_task_name, *args))

docker_exec('hoover-search', './manage.py', 'listcollections')
```

Reply:
At least for now, it's only used once and for combining 2 function calls in one line I think this is excessive. Regarding `shell`, we need it because it needs to be registered as a command.